CMakeLists.txt

```diff
@@ -201,9 +201,9 @@ elseif(GNU_GCC OR LLVM_CLANG)
         # TODO(rryan): macOS can use SSE3, and possibly SSE 4.1 once
         # we require macOS 10.12.
         # https://stackoverflow.com/questions/45917280/mac-osx-minumum-support-sse-version
-      elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "arm")
+      elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(aarch64|arm).*")
```

Comment:
Is the `.*` for stuff like `armv7h`, etc?

Reply:
-mfloat-abi=hard and -mfpu=neon is only valid for "arm" not for aarch64

See
https://developer.arm.com/documentation/dui0774/b/compiler-command-line-options/-mfloat-abi 
https://developer.arm.com/documentation/dui0774/b/compiler-command-line-options/-mfpu

Reply:
Yes. Might be not be needed for aarch64, but it doesn't hurt either.

Reply:
Maybe we should add a comment about if to avoid future confusion.

Reply:
Can you add a comment about your decision and than we can merge.

Reply:
Really needed?? Than I would need to duplicate it because the code is redundant. Just no. Alternatively I could restrict the `.*` suffix to `arm` if you prefer that.

Reply:
I don't mind if you keep the flags for arm64 or not. 

After these investigations, it turns out that the flag was just right before, but you was tempted to change it. This a clear hint that a comment is missing. Please leave a comment that this is not happen again. I don't like to repeat this cycle later, that's all. 
I also don mind if you duplicate. It would be sufficient in one place.